Cargo.toml

```diff
@@ -18,6 +18,7 @@ test = false # we have no unit tests
 doctest = false # and no doc tests
 
 [dependencies]
+measureme = { git = "https://github.com/rust-lang/measureme", rev = "501d6a3c192beee5e633a6c5f79130bedfdadcb5" }
```

Comment:
This isn't great, why do we need a git version of this crate?
I don't know the rustc repo policies here, but I doubt they will be very happy about such a dependency.

Reply:
This commit corresponds to https://github.com/rust-lang/measureme/pull/159 (Allow recording interval event wthout[sic] drop guard)

Reply:
Ah I see. Would it be possible to have a new release of that crate that includes this PR?

Reply:
I've updated to the latest release of `measureme`.